docs/OTCamera/gettingstarted/firstuse.md

```diff
@@ -1,4 +1,231 @@
+You can check the alignment with your mobile phone.
+It is best if two people do this.
+One person aligns the camera and the other checks the preview.
+
+Always wait a few seconds before adjusting OTCamera again, as a preview image is only generated every five seconds.
+
+!!! tip
+    We always align OTCamera so that no sky is visible.
+    Otherwise, glare can occur and traffic could no longer be seen.
+    In addition, there should be enough space in all directions, so that OTVision can recognize the trajectories of road users sufficiently.
+
+    Please always pay attention to these instructions especially if we should do the processing of the videos for you.
+
+If everything is aligned correctly, you can now lock OTCamera.
+
+Check all the values again on the preview page and make sure that everything is displayed correctly.
+Make sure that all screws are tightened and the fall protection is in place.
+
+## Dismounting
+
+When your measurement is finished, you can dismount OTCamera.
+
+Basically, just follow the installation steps in reverse order.
+When dismounting, make sure that nothing can drop (as always!).
+In particular, opening the screw on the ball head to loosen the mounting plate can cause OTCamera to fall down.
+Please take special care that this does not happen.
+
+When everything is dismounted and you are back on the ground, you can open the outer case and set the three right-hand switches (Boot, Wi-Fi, 24/7) to off.
+
+If OTCamera still has power, it will now shut down and indicate this with a rapidly flashing green LED.
+If the battery has run down in the meantime, OTCamera has already shut down on its own.
+Nevertheless, set all three switches to "0".
+
+!!! note
+    It is sufficient to set the "Boot" switch to "0".
+    However, we have made the experience that it is better to make sure that every switch is in the off position.
+    Switch on all the switches at the beginning of the measurement and switch them off at the end.
+    Then everything is back in the starting position for the next measurement.
+
+When the OTCamera is back in the office, charge the external USB-battery and the OTCamera itself.
+
+## Downloading videos
+
+After recharging the battery of the OTCamera you can download the recorded videos.
+You have two options:
+
+1. via Wi-Fi and SSH and
+2. to a USB flash drive
+
+### Wi-Fi
+
+Turn on OTCamera as described above.
+If OTCamera is connected to the Wi-Fi in your office, you can access via SSH using its IP address.
+This works the same way as during the installation process.
+
+Alternatively, you can establish a SSH connection using the access point of OTCamera.
+
+By default the recorded videos and logs are stored in the home directory in `~/videos`.
+You can download all videos via scp or rsync.
+
+### USB
+
+If you have an OTG USB adapter built into OTCamera, you can also have the videos copied to a flash drive.
```

Comment:
"OTG" is not explained on this page - so some customers maybe dont know it. Isnt "USB port" ok?